test/test-v8debugapi.ts

```diff
@@ -110,6 +110,44 @@ function validateBreakpoint(breakpoint: stackdriver.Breakpoint): void {
     breakpoint.stackFrames.forEach(validateStackFrame);
   }
 }
+describe('debugapi selection', function() {
+  const config: DebugAgentConfig = extend({}, defaultConfig, {
+    workingDirectory: __dirname,
+    forceNewAgent_: true
+  });
+  const logger = new common.logger({ levelLevel: config.logLevel } as any as LoggerOptions);
+  let warnIsCalled = false;
+  logger.warn = function(_s: string) {
+    warnIsCalled = true;
```

Comment:
We should make sure that `warn` was called and that the message supplied to `warn` was the expected message.